Add interface for teacher

djexvr opened the pull request.

djexvr committed:
✨(TeacherMeeting) add teacher interface

djexvr committed:
💄(home) differentiate interfaces background

The different interfaces are to similar. The different color background
differentiate the student and the teacher interfaces.

djexvr committed:
✨(StudentMeeting) custom jitsi toolbar

Add a custom toolbar to the Jitsi StudentMeeting

AladdinVR commented on box-ui/src/components/ts/Teacher/TeacherMeeting.tsx:
```diff
@@ -1,7 +1,108 @@
+    }, [navigate]);
+
+    return (
+        <div className='BoxMeeting'>
```
```suggestion
        <div className='TeacherMeeting'>
```

AladdinVR commented on box-ui/src/translations/en.json:
```diff
@@ -27,7 +27,12 @@
+        "teacherSpace": "Teacher Space",
+        "studentSpace": "Student Space",
+        "teacher": "Teacher",
+        "student": "Élève",
```
```suggestion
        "student": "Student",
```
English section here ^^

djexvr committed:
💄(Meeting) rename class based on usage

The class BoxMeeting is used in Box and Teacher interface and should be
renamed for clarity.

djexvr committed:
🌐(en) fix traduction error

djexvr committed:
🐛(StudentMeeting) fix to many prejoin menu

The prejoin menu to choose name and connection options was open at
each join. When the jitsi is minimize or maximize it reopens it which is
unwanted. It should only be open once when starting the meeting.
